projects/control-service/projects/helm_charts/pipelines-control-service/Chart.yaml

```diff
@@ -21,9 +21,9 @@ dependencies:
   repository: https://charts.bitnami.com/bitnami
   tags:
     - bitnami-common
-  version: 1.x.x
```

Comment:
what is this version about ?

Reply:
there were some chart incompatibilities with the postgresql chart so I updated it and it is ok now

Reply:
Cool